modules/document_repository/php/document_repository.class.inc

```diff
@@ -78,10 +78,9 @@ class Document_Repository extends \NDB_Menu_Filter
                           'v.Data_dir',
                           'v.Date_uploaded',
                          );
-
-        $openAccordion = $_GET['openAccordion'];
+        $openAccordion = isset($_GET['openAccordion']) ?? '';
         $this->tpl_data['openAccordion'] = $openAccordion;
-        $filtered = $_GET['filtered'];
+        $filtered = isset($_GET['filtered']) ?? '';
```

Comment:
this is what @johnsaigle did on minor: https://github.com/aces/Loris/pull/3603/commits/cdb3c94c510b69302347413c29f67f61c2c3bef0#diff-9df499b8cbcfa8bd78cf13ca7924b888